actors/builtin/power/power_test.go

```diff
@@ -105,6 +114,91 @@ func TestConstruction(t *testing.T) {
 		verifyEmptyMap(t, rt, st.PoStDetectedFaultMiners)
 		verifyEmptyMap(t, rt, st.CronEventQueue)
 	})
+
+	t.Run("ensure cronevents scheduled in null rounds are executed on next block", func(t *testing.T) {
+		rt := builder.Build(t)
+		actor.constructAndVerify(rt)
+
+		createMiner := func(owner, worker, miner, robust addr.Address, peer peer.ID) {
+			createMinerParams := &power.CreateMinerParams{
+				Owner:      owner,
+				Worker:     worker,
+				SectorSize: abi.SectorSize(int64(32)),
+				Peer:       peer,
+			}
+			initCreateMinerParams := &power.MinerConstructorParams{
+				OwnerAddr:  owner,
+				WorkerAddr: worker,
+				SectorSize: abi.SectorSize(int64(32)),
+				PeerId:     peer,
+			}
+
+			// owner send CreateMiner to Actor
+			rt.SetCaller(owner, builtin.AccountActorCodeID)
+			rt.SetReceived(abi.NewTokenAmount(1))
+			rt.ExpectValidateCallerType(builtin.AccountActorCodeID, builtin.MultisigActorCodeID)
+
+			createMinerRet := &power.CreateMinerReturn{
+				IDAddress:     miner,  // miner actor id address
+				RobustAddress: robust, // should be long miner actor address
+			}
+			createMinerBuf := new(bytes.Buffer)
+			err := createMinerParams.MarshalCBOR(createMinerBuf)
+			require.NoError(t, err)
+
+			initCreateMinerBuf := new(bytes.Buffer)
+			err = initCreateMinerParams.MarshalCBOR(initCreateMinerBuf)
+			require.NoError(t, err)
```

Comment:
Break the creation and serializatio of initCreateMinerParams into a function, because it will be re-used a lot.